Opentrons/buildroot
# feat(build): Add a/v dependencies

- ffmpeg needed some extra codecs to do software scaling
- mpg123 needed alsa userspace libs
- need to actually enable the soc audio support in config.txt

Closes Opentrons/opentrons#2875
Closes https://github.com/Opentrons/opentrons/issues/2874
## Testing
 - Install this on a buildroot robot. You need to either install by flashing the sd card since this changes config.txt or just edit it by doing `mkdir /mnt/boot; mount /dev/mmcblk0p1 /mnt/boot`; and then editing `/mnt/boot/config.txt` to add `dtparam=audio=on` at the end, then restart
- Make the robot sing with an mp3: `mpg123 /path/to/audio.mp3`
- Make the robot record video: `ffmpeg -video_size 230x240 -i /dev/video0 -t 00:00:01 /path/to/output.mp4` and check that it actually plays
